docs/code_contribution_guidelines.md

```diff
@@ -27,7 +27,7 @@ Developing cryptocurrencies is an exciting endeavor that touches a wide variety
 of areas such as wire protocols, peer-to-peer networking, databases,
 cryptography, language interpretation (transaction scripts), adversarial
 threat-modeling, and RPC systems. They also represent a radical shift to the
-current fiscal system and as a result provide an opportunity to help reshape
+current financial system and as a result provide an opportunity to help reshape
```

Comment:
why the change?

Reply:
I wasn't sure how LN is related to fiscal systems (gov revenue & expenditure, taxes, etc.)? Now on second thought, rewording it to financial makes it tautological.

So I am thinking `monetary` system might be a better rewording as Bitcoin **is** a radically different monetary network/system vis-a-vis the existing one.

Reply:
@yyforyongyu Let me know what you think, I just swapped out `fiscal` for `monetary` in f1ac44dd01615370eb44388f750f1c70d77e5ed0 instead of `financial` for the reasons I mentioned above.

Reply:
IMO, the gov will be forced to accept LN and thus one day build its fiscal policy on top of LN, mostly taxation. `monetary` is more about managing the supply/demand of money. To me it's like two different ways for the gov to manage the financial system, and I'm ok with either word.

Strictly speaking, I would not consider this a typo fix as it's related to semantics. However since it's not related to code change so I think it's ok.

Reply:
Ah, fair enough. Thanks for clarifying.

I'll update the commit message to something more general to reflect your point about it not being a typo. This way, it removes any confusion in the codebase down the line 👍🏽